src/Passes/manage

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
```

Comment:
This mechanism for running tests and verifying the code seems very Linux specific. Is development of passes on Windows not supported by this infrastructure yet?

Reply:
That's a question to answer with a Windows machine handy. :) If the infrastructure for custom passes works well on Windows, then we should certainly keep the scripts working for it as well. If not, we need to make a call regarding our pass infrastructure.

Reply:
Given that it is a Python script, you should be able to run this on a Windows machine. This is not too different to how frameworks like `Django` works and those runs on Windows for sure :). That said, the CI tool does rely on `shutil` and `subprocess` which may act different on different OSes. Finally, we will probably need to run CI inside a docker container regardless to ensure that `clang-format` is reproducible across all systems.

I am happy to set up a Windows VM, to test and document Windows behaviour if this is considered high priority.

Reply:
We discussed this a bit offline, and need to do a separate PR and discussion around the build. Right now the CI only builds the compiler repo on Windows, which made sense when the repo was managed code only. Now that more native code components are being introduced, we should update the CI to be cross-platform accordingly.